.gitignore

```diff
@@ -30,3 +30,6 @@ docs/
 
 # Emacs
 *~
+
+# Visual Studio Code
+.vscode
```

Comment:
Please add a newline at the end of the file
Also, since it's a directory, use `.vscode/`

Reply:
Yes, I'll modify it.